### PR TITLE
Commet with foss2016a

### PR DESCRIPTION
**Pull request by RvDijk**
Source has to be downloaded manually. Source is renamed to Commet-20160422.zip

### CODE REVIEW

File: easybuild/easyconfigs/c/Commet/Commet-20160422-foss-2016a-Python-2.7.11.eb [hunk 1]
```diff
@@ -0,0 +1,31 @@
+easyblock = 'MakeCp' 
+
+name = 'Commet' 
+version = '20160422' 
+
+homepage = 'https://colibread.inria.fr/software/commet/' 
+description = """ COMMET (“COmpare Multiple METagenomes”) provides a global similarity overview between all datasets of a large metagenomic project.
```
Comment (boegel) [inline easybuild/easyconfigs/c/Commet/Commet-20160422-foss-2016a-Python-2.7.11.eb]: this looks longer than 120 chars? if so, break it up in two lines please

File: easybuild/easyconfigs/c/Commet/Commet-20160422-foss-2016a-Python-2.7.11.eb [hunk 2]
```diff
@@ -0,0 +1,31 @@
+easyblock = 'MakeCp' 
+
+name = 'Commet' 
+version = '20160422'
```
Comment (boegel) [inline easybuild/easyconfigs/c/Commet/Commet-20160422-foss-2016a-Python-2.7.11.eb]: mention the commit this relates to?

File: easybuild/easyconfigs/c/Commet/Commet-20160422-foss-2016a-Python-2.7.11.eb (hunk 1 above)
Comment (boegel) [inline easybuild/easyconfigs/c/Commet/Commet-20160422-foss-2016a-Python-2.7.11.eb]: also, please replace the unicode `"` characters with regular ones, while you're at it

File: easybuild/easyconfigs/c/Commet/Commet-20160422-foss-2016a-Python-2.7.11.eb [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+toolchain = {'name': 'foss', 'version': '2016a'}
+
+# there is no tarball provided, only git checkout through: 
+# https://github.com/pierrepeterlongo/commet/archive/master.zip
+sources = [SOURCE_ZIP] 
+files_to_copy = ["bin","doc","include","ABCDE_bench",(["Commet.py"],'bin')] 
+
+pythonversion = '2.7.11' 
+versionsuffix = '-Python-%(pythonversion)s'
```
Comment (boegel) [inline easybuild/easyconfigs/c/Commet/Commet-20160422-foss-2016a-Python-2.7.11.eb]: please use the `%(pyver)s` template provided by framework:

```python
version = ...
versionsuffix = '-Python-%(pyver)s'
...
dependencies = [
    ('Python', '2.7.11'),
]
```

File: easybuild/easyconfigs/c/Commet/Commet-20160422-foss-2016a-Python-2.7.11.eb [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+files_to_copy = ["bin","doc","include","ABCDE_bench",(["Commet.py"],'bin')] 
+
+pythonversion = '2.7.11' 
+versionsuffix = '-Python-%(pythonversion)s' 
+
+dependencies = [
+    ('Python', pythonversion), 
+] 
+
+checksums = ['5fe63369ec3722e9e758b7bc1dcdfef3']
```
Comment (boegel) [inline easybuild/easyconfigs/c/Commet/Commet-20160422-foss-2016a-Python-2.7.11.eb]: move this up right below source please, and definitely mention to which commit this relates to

File: easybuild/easyconfigs/c/Commet/Commet-20160422-foss-2016a-Python-2.7.11.eb [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+versionsuffix = '-Python-%(pythonversion)s' 
+
+dependencies = [
+    ('Python', pythonversion), 
+] 
+
+checksums = ['5fe63369ec3722e9e758b7bc1dcdfef3'] 
+
+sanity_check_paths = {
+    'files': ["bin/%s" % binfile for binfile in ["Commet.py","bvop","compare_reads","extract_reads","filter_reads","index_and_search"]],
```
Comment (boegel) [inline easybuild/easyconfigs/c/Commet/Commet-20160422-foss-2016a-Python-2.7.11.eb]: please use space after `,`

also, line too long?

**Comment by boegel** [comment]
Jenkins: ok to test

File: easybuild/easyconfigs/c/Commet/Commet-20160422-foss-2016a-Python-2.7.11.eb (hunk 2 above)
Comment (RvDijk) [inline easybuild/easyconfigs/c/Commet/Commet-20160422-foss-2016a-Python-2.7.11.eb]: Do you have any explanations on how to do this?

Comment (RvDijk) [inline easybuild/easyconfigs/c/Commet/Commet-20160422-foss-2016a-Python-2.7.11.eb]: @boegel should it be 4ef0705 as shown on: https://github.com/pierrepeterlongo/commet/commits/master ?

Comment (boegel) [inline easybuild/easyconfigs/c/Commet/Commet-20160422-foss-2016a-Python-2.7.11.eb]: yeah, basically do something like:

```python
version = '20150415'
commit = '4ef0705'
...
source_urls = ['https://github.com/pierrepeterlongo/commet/archive']
sources = ['%s.tar.gz' % commit]
```

note that I'm using the date that corresponds to that commit

**Comment by boegel** [comment]
@RvDijk some more (minor) style fixes in https://github.com/rug-cit-ris/easybuild-easyconfigs/pull/1

If you merge that PR, this PR will be updated and should be good to go (if the test reports are positive).

**Comment by RvDijk** [comment]
@boegel I accidentally fixed the things you pointed out into a new commit instead of merging, I hope this is fine too otherwise I will undo the commit such that your PR can be merged.

File: easybuild/easyconfigs/c/Commet/Commet-20150415-foss-2016a-Python-2.7.11.eb [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+
+dependencies = [
+    ('Python', '2.7.11'), 
+] 
+
+files_to_copy = ["bin","doc","include","ABCDE_bench",(["Commet.py"],'bin')] 
+
+sanity_check_paths = {
+    'files': ['bin/%s' % binfile for binfile in ['Commet.py', 'bvop', 'compare_reads',
+	                                             'extract_reads', 'filter_reads', 'index_and_search']],
```
Comment (boegel) [inline easybuild/easyconfigs/c/Commet/Commet-20150415-foss-2016a-Python-2.7.11.eb]: @RvDijk please remove the `<tab>` character here in the front  (we will start enforcing the lack of tabs soonish)

**Comment by boegel** [comment]
@RvDijk I don't mind which commit is used, I'll close my PR

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2124.delcatty.os - Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/36f718dd8f0c90e5d67ea54dfb8b3782 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2465.golett.os - Linux centos linux 7.2.1511, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/2f4434518f063f22e024fb3631d4b4be for a full test report.

**Comment by boegel** [comment]
lgtm

**Comment by boegel** [comment]
Going in, thanks @RvDijk!

**Pull request merged**